### DISCUSSION TITLE
Steam Config

### DISCUSSION THREAD

**Discussion by jvle** [Zer0-bit/gaggiuino#326] [Q&A] [answered]
eco vs non-eco versions:
How do I know if my GCP is a eco or non-eco version?  
Of did you really mean the connection is just ech vs non-eco wiring way?
https://gaggiuino.github.io/#/gcp/gaggia-classic-pro-new-classic?id=_204-steam-config

Thanks,
Jason

**Comment by Zer0-bit** [comment, marked as answer]
The ECO GCP has an auto shut off timer and the power switch is of a bounce back type.
